hub/service/sweep_service.cc

```diff
@@ -286,6 +287,11 @@ bool SweepService::doTick() {
         hubInputs.clear();
         requiredOutput = 0;
         withdrawals.clear();
+      } else {
+        std::vector<db::TransferInput> minimalVecOfInputs;
+        getVecOfMinSizeWithSumNotLessThan(missing, hubInputs,
```

Comment:
You want to do the `getVecOfMinSizeWithSumNotLessThan` in `helper<C>::getHubInputsForSweep` instead of here.
The `hubInputs` are already prefiltered when they reach this point.